README.md

```diff
@@ -313,6 +324,9 @@ This may seem foreign to some Stata and SAS programmers but it has several advan
 
 -   All final files should be in the `.csv` format. The files should be delimited with a comma.
 -   Do not open and save any `.csv` files in Microsoft Excel. Excel defaults lead to analytic errors.
+-   Final metric files should have a row for all geographies that are included in the all files combined data set 
+       -   For county files with 2020 data and onward this will be 3,143 observations.
+       -   For place files this will be 386 observations.
```

Comment:
I see a place crosswalk "census_place_2020poulation_allFIPS.csv" that has more than 386 observations. It has 487. Where did 386 come from?

Reply:
No idea why i had 386 - though I do think the correct number is 486 (which we note in a section above), as I belive your 487 includes the headers